test_scripts/MobileProjection/Defects/3139_01_Stop_streaming_gracefully_for_1st_app_if_2nd_is_activated_and_starts_streaming.lua

```diff
@@ -0,0 +1,131 @@
+  log("SDL->HMI ", "SDL.ActivateApp(2)")
+  ts("ActivateApp")
+  common.hmi.getConnection():ExpectResponse(cid)
+  :Do(function()
+      log("SDL->HMI ", "SUCCESS: SDL.ActivateApp")
+    end)
+
+  common.mobile.getSession(2):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "FULL", videoStreamingState = "STREAMABLE" })
+  :Do(function()
+      log("SDL->App2", "OnHMIStatus(FULL)")
+      ts("OnHMIStatus_Full", "app2")
+    end)
+
+  common.mobile.getSession(1):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", videoStreamingState = "NOT_STREAMABLE" })
+  :Do(function()
+      log("SDL->App1", "OnHMIStatus(NOT_STREAMABLE)")
+      ts("OnHMIStatus_Not_Streamable", "app1")
+    end)
+
+  common.hmi.getConnection():ExpectNotification("Navigation.OnVideoDataStreaming", { available = false })
+  :Do(function()
+      log("SDL->HMI ","Navi.OnVideoDataStreaming(false)")
+      ts("Navi.OnVideoDataStreaming", "hmi")
+    end)
+
+  common.hmi.getConnection():ExpectRequest("Navigation.StopStream", { appID = common.app.getHMIId(1)})
+  :Do(function(_, data)
+      log("SDL->HMI ","Navi.StopStream")
+      ts("Navi.StopStream", "hmi")
+      common.hmi.getConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+      log("HMI->SDL ","SUCCESS: Navi.StopStream")
+    end)
+
+  common.mobile.getSession(1):ExpectEndService(11)
+  :Do(function()
+      log("SDL->App1", "EndService(VIDEO)")
+      ts("EndService", "app1")
+      common.mobile.getSession(1):SendEndServiceAck(11)
+      log("App1->SDL", "EndServiceAck")
+      common.stopStreaming(1)
+    end)
+
+  common.mobile.getSession(1):ExpectNotification("OnAppInterfaceUnregistered")
+  :Do(function()
+      log("SDL->App1", "OnAppInterfaceUnregistered")
+    end)
+  :Times(0)
+
+  common.hmi.getConnection():ExpectNotification("BasicCommunication.OnAppUnregistered")
+  :Do(function()
+      log("SDL->HMI ","BC.OnAppUnregistered(App1)")
+    end)
+  :Times(0)
+
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions, { { StopStreamingTimeout = 3000 } })
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App 1", common.registerApp, { 1 })
+runner.Step("Register App 2", common.registerApp, { 2 })
+runner.Step("Activate App 1", common.activateApp, { 1 })
+
+runner.Title("Test")
+runner.Step("App 1 start video streaming", common.startStreaming, { 1, 11 })
+runner.Step("App 1 deactivated", common.deactivateApp, { 1 })
+runner.Step("App 2 activated, App 1 is not unregistered", activateApp2)
+runner.Step("App 2 start video streaming", common.startStreaming, { 2, 11 })
+
+runner.Step("Verify timeout for OnHMIStatus_Not_Streamable", common.checkTimeout,
+  { "ActivateApp", "OnHMIStatus_Not_Streamable", 0 })
+runner.Step("Verify timeout for EndService", common.checkTimeout,
+  { "ActivateApp", "EndService", 0 })
```

Comment:
@GetmanetsIrina Since approach with single timer has been selected please notice the following for `OnHMIStatus_Not_Streamable` and `EndService` messages:
1) There is not delay between them
2) Their order can be any